src/stream/sink/image_sink.rs

```diff
@@ -9,16 +12,21 @@ use image::FlatSamples;
 use gst::prelude::*;
 
 use super::SinkInterface;
-use crate::{stream::pipeline::PIPELINE_SINK_TEE_NAME, video::types::VideoEncodeType};
+use crate::{stream::pipeline::runner::PipelineRunner, video::types::VideoEncodeType};
 
 #[derive(Debug)]
 pub struct ImageSink {
     sink_id: uuid::Uuid,
+    pipeline: gst::Pipeline,
     queue: gst::Element,
-    transcoding_elements: Vec<gst::Element>,
+    proxysink: gst::Element,
+    _proxysrc: gst::Element,
```

Comment:
Clippy was mumbling it wasn't being accessed from the struct, I can check again.

Reply:
I checked, and since I don't really need to do anything with it during the `unlink`, it's unused. We could just get rid of it since it's not being used.